simphony_mayavi/plugins/tabbed_panel_collection.py

```diff
@@ -3,12 +3,42 @@
 
 
 class TabbedPanelCollection(HasTraits):
+    """Collect a list of Traits object and display each
+    of them as a tab in a tabbed notebook using ListEditor
+    """
 
+    # List of Traits object to be displayed in the notebook
     panels = List
+
+    # The Traits object currently selected
     selected_panel = Any
 
     @classmethod
     def create(cls, **kwargs):
+        """Create a TabbedPanelCollection containing the given
+        Traits objects.
+
+        Optional keyword arguments
```

Comment:
This should be 
```
Arguments
--------------
**kwargs : 
     The keys in the keyword arguments are used to define
     attributes of the TabbedPanelCollection
     so that the Traits object can be retrieved easily.
 
     As with any keyword arguments, the order of the keys
     is lost.
```